guides/v2.3/graphql/reference/store-config.md

```diff
@@ -1,38 +1,80 @@
 ---
 group: graphql
-title: storeConfig endpoint
+title: Store endpoint
 ---
 
-The `storeConfig` endpoint returns information about a store's configuration, including its locale, currency codes, and the secure and unsecure URLs.
+The `Store` endpoint defines information about a store's configuration.
```

Comment:
Add a link to https://devdocs.magento.com/guides/v2.3/graphql/send-request.html with an explanation that the Store header allows you to query a non-default store.

Reply:
fixed